blogs/2021/01/ext-bisect.md

```diff
@@ -0,0 +1,40 @@
+# “Just like git-bisect, but for VS Code extensions.”
```

Comment:
Uhm not fully convinced about this title. If someone doesn't know what git bisect is, this doesn't make much sense.

How about something like "Resolving extension issues with bisect" or something like that?

Reply:
Agree with this. As I understand it, _bisect_ in git refers to quickly finding the commit that introduced a bug. So proposed...

1. Introducing Bisect - easily identify VS Code extensions that are causing problems